tiffslide/tiffslide.py

```diff
@@ -14,6 +14,7 @@
 from typing import Mapping
 from typing import overload
 from warnings import warn
+import xmltodict
```

Comment:
Instead of adding xmltodict as a dependency, I think it's better to roll our own minimal version of this:

```python
def _xml_to_dict(xml: str) -> dict:
    def _to_dict(e):
        tag = e.tag[e.tag.find("}") + 1:]
        d = {f"@{k}": v for k, v in e.attrib.items()}
        for c in e:
            key, val = _to_dict(c).popitem()
            if key not in d:
                d[key] = val
            elif not isinstance(d[key], list):
                d[key] = [d[key], val]
            else:
                d[key].append(val)
        if e.text and e.text.strip():
            if d:
                d["#text"] = e.text
            else:
                d = e.text
        return {tag: d}

    x = ElementTree.fromstring(xml)
    return _to_dict(x)
```

This should do the trick.